content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.rst

```diff
@@ -12,7 +12,7 @@ Inventory Adjustments page
 ==========================
 
 To view the *Inventory Adjustments* page, navigate to :menuselection:`Inventory app --> Operations
---> Inventory Adjustments`.
+--> Physical Inventory`.
```

Comment:
This line is great! But I'm gonna leave this comment here because I can't comment on lines 24 - 58. 

The existing documentation covers most fields, but in 17.0, they made some more column options.
![image](https://github.com/user-attachments/assets/49080014-b0cc-43e6-a5fa-9dd3c719e5fe)

Our documentation should also define what information each of these fields display. Can you do that for me, when implementing my suggestions? (:

For the sake of organizing more relevant information higher in the section, my suggestion is this:
- display the definitions of the columns available in the Inventory Adjustment table, first (you already have this :smile: )
- Then, in a note admonition block, write something like, "The fields below are optional fields that can be made visible by clicking the :icon:`oi-settings-adjust` :guilabel:`(settings)` icon in the top-right of the :guilabel:`Inventory Adjustments` page. 
- define each of the optional fields (let me know if you need help finding information about any of these fields! I suggest trying to look them up in the documentation first, and if you absolutely can't find anything, lmk :heart:)